drivers/wd/README.md

```diff
@@ -0,0 +1,35 @@
+Getting started with the Webdav driver
+======================================
+
+The driver handles the following options:
+
+- hostname (no default value)
+- username (default value is "")
+- password (default value is "")
+- changes_timer (default value is 60)
+
+An example of setup.yml which can be used to store files on [box](http://box.com)
+
```

Comment:
Could you add some example on how to start a server, at least for debugging purpose ?